contributed_definitions/nyaml/NXmpes_liquid.yaml

```diff
@@ -98,119 +21,57 @@ NXmpes_liquid(NXmpes):
                 doc: "Is '.', since beam axis is assumed to be horizontal (perpendicular to gravity)."
               \@transformation_type(NX_CHAR):
                 enumeration: ["rotation"]
-              \@units: NX_ANGLE
+              unit: NX_ANGLE
               \@vector(NX_NUMBER):
             rot_y(NX_NUMBER):
```

Comment:
Here the question is whether the users should have to follow a certain transformation chain. In principle they could use any chain they want here. Do you have a certain reason why it is better to demand this specific kind of transformation chain?

Reply:
I think this can be open to the user and inherited from the MPES definition. From my POV, we do not require anything specific for defining the detection system / analyzer. However, if a flatjet is present, its orientation (surface normal) is defined with respect to the lens axis. However, I believe this is a similar situation with ARPES, where the surface normal is defined for sample rotation.

Also, this might change in the future, if people want to specify something like differential pumping stages / orifices, which are an integral component of liquid PES experiments as compared to UHV PES. For now, I don't see such a requirement, though.